doc/v2.md

```diff
@@ -0,0 +1,20 @@
+## Guide for Migration from v1 to v2
+
+`v2` contains lots of refactoral changes to improve the performance and configuration by introducing [`@logdna/logger`](https://www.npmjs.com/package/@logdna/logger), [`@logdna/env-config`](https://www.npmjs.com/package/@logdna/env-config), and [`eslint-config-logdna`](https://www.npmjs.com/package/eslint-config-logdna). There are several changes in naming and handling environment variables as well. All environment variables are documented [here](./env.md)
+
+### Renamed Variables
+* `LOGDNA_KEY` has been replaced with [`INGESTION_KEY`](./env.md#ingestion_key)
+* `LOGDNA_HOSTNAME` has been replaced with [`HOSTNAME`](./env.md#hostname)
+* `LOGDNA_TAGS` has been replaced with [`TAGS`](./env.md#tags)
+* `LOGDNA_URL` has been split into multiple new variables: [`INGESTION_ENDPOINT`](./env.md#ingestion_endpoint), [`INGESTION_HOST`](./env.md#ingestion_host), [`INGESTION_PORT`](./env.md#ingestion_port), and [`SSL`](./env.md#ssl).
+* `LOGDNA_BATCH_INTERVAL` has been replaced with [`FLUSH_INTERVAL`](./env.md#flush_interval)
+* `LOGDNA_BATCH_LIMIT` has been replaced with [`FLUSH_LIMIT`](./env.md#flush_limit)
+
+### Removed Variables
+* `LOGDNA_FREE_SOCKET_TIMEOUT`, `LOGDNA_MAX_REQUEST_RETRIES`, `LOGDNA_REQUEST_RETRY_INTERVAL`, and `LOGDNA_MAX_REQUEST_TIMEOUT` have been removed since the connection gets handled within the logger client
+* `LOGDNA_MAX_LINE_LENGTH` has been removed since a line length gets handled in the server-side
+* `LOGDNA_EVENTLOG` and `LOGDNA_FILELOG` have been removed since `v2` streams only the logs from the files within the specified bucket. You can use `AWS CloudTrail Event Logging` to stream Object-level operations from `S3` buckets. Check out [this article](https://docs.amazonaws.cn/en_us/AmazonS3/latest/userguide/enable-cloudtrail-logging-for-s3.html)
```

Comment:
Same as above with "were replaced".
Add period (.) to end of sentence "Check out this article."
Consider changing "Check out this article." to "For more information, refer to this article."

Reply:
Done!